Edit javascript codebyte1 (#2245)
Codecademy/docs: content/javascript/concepts/dates/terms/getDate/getDate.md

  - 'paths/front-end-engineer-career-path'
---

The **`.getDate()`** method returns the day of the month.

## Syntax

> ```js

Reviewer: ```suggestion
```shell
```

myDate.getDate();
```

> The return value of `getDate()` is an integer number between `1` and `31` representing the day of a month.

Reviewer: ```suggestion
The return value of `.getDate()` is an integer number between `1` and `31` representing the day of a month.
```

// Output: It is the 10th, or later!
```

## Codebyte Example

> The example below is runnable, and first, it defines a variable `myDate` with a given new `Date`, then uses the `getDate()` method to store the day from that date into a variable called `payDay`. Then `console.log` prints this day to the screen:

Reviewer: ```suggestion
The example below is runnable, and assigns the variable `myDate` to a new given `Date`, then the result of the `.getDate()` method is stored in `payDay`. Finally, a `console.log` call prints this value to the screen:
```

Reviewer: Generally, I think these example introductions should be minimal.
